helm/ds/values.yaml

```diff
@@ -14,7 +14,8 @@ useDefaultSecrets: true
 # The setting below changes the base dn for the o=userstore backend only!
 # We strongly suggest that you *NOT* change the default. There is very little value in
 # having the basedn reflect a unique organizational structure.
-baseDN: "o=userstore"
+#baseDN: "o=userstore"
```

Comment:
The kind of contradicts the comment on lines 14-16. If you can't use o=userstore, please change the comment.